backend/src/docs/v1.api-spec.yaml

```diff
@@ -277,6 +309,21 @@ components:
       type: openIdConnect
       openIdConnectUrl: 'https://example.com/.well-known/openid-configuration'
   schemas:
+    AcronymDetail:
+      type: object
+      properties:
+        acronym:
+          type: string
+          example: MSSC
+        name:
+          type: string
+          example: Messaging Service Showcase
+        permissionWebade:
```

Comment:
Do we want to consider just a flags field and do a bitwise comparison for permissions instead of adding individual flag fields as they come up?

Reply:
Discussed this morning